subjects/devops/orchestrator/README.md

```diff
@@ -0,0 +1,145 @@
+## Orchestrator
+
+![Orchestrator](pictures/Orchestrator.jpg)
+
+### Objectives
+
+In this project, you will deploy a microservices architecture on Kubernetes, You will gain experience with key technologies and concepts such as Kubernetes architecture, deployments, services, ingresses, and API gateways. Additionally, this project will provide you with an opportunity to practice DevOps skills such as containerization, continuous integration, and deployment (CI/CD), and infrastructure as code (IaC) using Kubernetes manifests. By completing this project, you will have a solid understanding of microservices architecture and the tools and techniques used to deploy and manage such systems using Kubernetes.
+
+### Tips
+
+- Spend time on the theory before rushing into the practice.
+- Read the official documentation.
+- You must understand the k8s components
+
+> Any lack of understanding of the concepts of this project may affect the difficulty of future projects, Take your time to understand all concepts.
+
+> Be curious and never stop searching!
+
+### Architecture
+
+![Architecture](pictures/Architecture.png)
+
+You have to deploy this microservices architecture in a k3s cluster consisting of the following components:
+
+- `inventory-database container` is a PostgreSQL database server that contains your inventory database, it must be accessible via port `5432`.
+- `billing-database container` is a PostgreSQL database server that contains your billing database, it must be accessible via port `5432`.
+- `inventory-app container` is a Node.Js server that contains your inventory-app code running and connected to the inventory database and accessible via port `8080`.
+- `billing-app container` is a Node.Js server that contains your billing-app code running and connected to the billing database and consuming the messages from the RabbitMQ queue, and it can be accessed via port `8080`.

```

Comment:
idem

Reply:
I can't understand

Reply:
idem means `the same`, in that sence here too `Node.Js` -> `Node.js`, in any case I saw you already applied the changes ;)